eng/pipelines/ci-cadl-java.yaml

```diff
@@ -71,14 +71,14 @@ jobs:
       - script: |
           node node_modules/\@azure-tools/cadl-ranch/dist/cli/cli.js serve node_modules/\@azure-tools/cadl-ranch-specs/http/ --coverageFile ./cadl-ranch-coverage-java.json &
```

Comment:
do we want do change the file name to ci-typespec-java.yaml?

Reply:
renamed, hope CI still works